paymentsheet/src/main/java/com/stripe/android/paymentsheet/BasePaymentMethodsListFragment.kt

```diff
@@ -75,89 +96,54 @@ internal abstract class BasePaymentMethodsListFragment(
                 color = Color(appearance.getColors(context.isSystemDarkTheme()).appBarIcon),
                 fontFamily = appearance.typography.fontResId
             )
-            isVisible = adapter.hasSavedItems()
-        }
-    }
-
-    @Deprecated("Deprecated in Java")
-    override fun onOptionsItemSelected(item: MenuItem): Boolean {
-        return when (item.itemId) {
-            R.id.edit -> {
-                isEditing = !isEditing
-                true
-            }
-            else -> super.onOptionsItemSelected(item)
         }
     }
 
-    override fun onSaveInstanceState(outState: Bundle) {
-        outState.putBoolean(IS_EDITING, isEditing)
-        super.onSaveInstanceState(outState)
-    }
```

Comment:
Getting rid of this here while we’re re-evaluating how we want to handle process death. Also not convinced that restoring the current edit state is super useful.